homework/nwd-nnw/nwdNww.hpp

```diff
@@ -1,11 +1,22 @@
 #pragma once
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+   int temp;
+
+   while (rhs!=0) {
+       temp = rhs;
+       rhs = lhs%rhs;
+       lhs = temp;
+   }
+   if (lhs<0){
+       return lhs*(-1);
+   }
```

Comment:
Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..